tests/cli/cdk_smoke_tests/cdk-multi-partition-consumer.bats

```diff
@@ -113,55 +113,3 @@ EOF
     assert_success
 }
 
-@test "Topic with 3 partitions. Consumer reads 2 partitions" {
```

Comment:
Shouldn't  this be migrated rather than delete?

Reply:
The reason this test is being deleted is that the new consumer API doesn't support several partitions, only one or all. But, actually, supporting it is relatively cheap. I am going to add support for it, and we won't need to delete tests.

Reply:
this test is kept without changes